src/test/ParkVehicleTest.java

```diff
@@ -0,0 +1,30 @@
+package test;
+
+import main.models.terminal.Terminal;
+import main.models.terminal.TerminalWorker;
+import main.models.vehicles.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParkVehicleTest {
```

Comment:
Since you are testing the Terminal in this case, it would be more appropriate to call this class `TerminalTest` or something like that :)

Reply:
Done.